CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Master (Unreleased)
 
+- Add `rubocop-capybara` version constraint to prevent sudden cop enabling when it hits 3.0. ([@pirj])
+
 ## 2.18.0 (2023-01-16)
 
 - Extract Capybara cops to a separate repository. ([@pirj])
```

Comment:
Could we address https://github.com/rubocop/rubocop-rspec/commit/eed7439be0c90c7d79e264240f91ad0a924d5a80#r96342930 too with e.g.
```suggestion
- Extract Capybara cops to a separate repository. The `rubocop-capybara` repository is a dependency of `rubocop-rspec` and the Capybara cops are aliased (`RSpec/Capybara/Foo` == `Capybara/Foo`) until v3.0 is released, so the change will be invisible to users until then. ([@pirj])
```